fractal-puppy/Engine
# Enemy scripts VS3 2.0

Updated basic and hidden enemy scripts and added first version of range enemy.

To test:
Basic enemy: Open scene "Level0-TheGraveyard", generate the nav mesh. Hit play and attack the enemy located after graveyard door. The enemy should follow you and attack, when you attack the enemy the number with the damage should pop up and after 2 hits the enemy should die.

Hidden enemy: Open scene "HiddenEnemyAnimated", hit play and the enemy should follow this behaviour https://drive.google.com/drive/folders/1ZVkjjNBx_7R0dYOCQ1DZbnbM27AcSgHQ. To move the player use the editor.

![Hidden enemy animated](https://user-images.githubusercontent.com/17391875/58470238-870acc80-8141-11e9-93b1-aaf6407db003.gif)

Hidden enemy: Open scene "RangeEnemyAnimated", hit play and the enemy should follow this behaviour https://drive.google.com/drive/folders/1AmDcEM9BDA8y6Nt_04cI0uaTmSSl2kws. To move the player use the editor.
